code/extensions/CwpSiteSummaryExtension.php

```diff
@@ -20,4 +19,27 @@ public function updateVersionModules(&$modules)
         $modules = ['cwp/cwp' => 'CWP'] + $modules;
         $modules['silverstripe/cms'] = 'SilverStripe CMS';
     }
+
+    /**
+     * Updates the dropdown filter used to filter supported packages by renaming the labels (replaces the existing
+     * filter options)
+     *
+     * @param GridFieldDropdownFilter $dropdownFilter
+     */
+    public function updateDropdownFilterOptions($dropdownFilter)
+    {
+        $dropdownFilter->removeFilterOption('supported');
+        $dropdownFilter->removeFilterOption('unsupported');
+
+        $dropdownFilter->addFilterOption(
+            'supported',
+            _t(__CLASS__ . 'filterSupported', 'CWP recipe modules'),
```

Comment:
The translation key would evaluate as `CwpSiteSummaryExtensionfilterSupported` - needs a `.`

Reply:
👍 Fixed